cmd/prepare-storage.go

```diff
@@ -131,20 +131,20 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 		return WaitForConfig
 	}
 
-	quorum := diskCount/2 + 1
+	readQuorum := diskCount / 2
 	disksOffline := errMap[errDiskNotFound]
 	disksFormatted := errMap[nil]
 	disksUnformatted := errMap[errUnformattedDisk]
 	disksCorrupted := errMap[errCorruptedFormat]
 
 	// No Quorum lots of offline disks, wait for quorum.
-	if disksOffline >= quorum {
+	if disksOffline >= readQuorum {
```

Comment:
If we have 2 offline disks of 4 total disks, disksOffline will be 2 and readQuorum will be also 2, doesn't this mean that we will never be able to run Minio, right?